docs/user-guide/scheduled-jobs.md

```diff
@@ -19,6 +19,9 @@ A _Scheduled Job_ manages time based [Jobs](/docs/user-guide/jobs/), namely:
 One ScheduledJob object is like one line of a _crontab_ (cron table) file. It runs a job periodically
 on a given schedule, written in [Cron](https://en.wikipedia.org/wiki/Cron) format.
 
+NOTE: The question mark (`?`) in the schedule has the same meaning as an asterisk `*`,
+that is, it stands for any of available value for a given field.
```

Comment:
It's just that I didn't realize `*/1 * * * ?` and `*/1 * * * *` are exactly the same until I saw this paragraph. 

When I first saw `*/1 * * * ?` I immediately assumed that `?` and `*` are somewhat semantically different.